dotcom-rendering/src/web/components/Caption.tsx

```diff
@@ -232,6 +234,7 @@ export const Caption = ({
 	shouldLimitWidth = false,
 	isOverlaid,
 	isLeftCol,
+	isLightbox,
```

Comment:
I'm not saying you should rewrite this component but there's an awful lot of boolean props here and maybe having `CoreCaption` contain the shared logic and then breaking out styles to `LightboxCaption`, `OverlayCaption`, `ArticleCaption`, etc.

See, I can make comment like this and now I feel totally fine about adding yet another boolean prop to Caption 😇

Reply:
Sorry, I forgot about this but remembered just now when opening this file for something else.

I have [now created](https://github.com/guardian/dotcom-rendering/pull/7129/commits/94a5260a06014833204291709cb8dd3dde1f794a) `LightboxCaption`